src/Cafe/GraphicPack/GraphicPack2PatchesApply.cpp

```diff
@@ -125,14 +125,14 @@ double _cbResolveConstant(std::string_view varname)
 		if (idxDot != std::string_view::npos)
 		{
 			std::string moduleName = importName.substr(0, idxDot);
-			std::string functionName = importName.substr(idxDot + 1);
 			uint32 rplHandle = RPLLoader_GetHandleByModuleName(moduleName.c_str());
 			if (rplHandle == RPL_INVALID_HANDLE)
 			{
 				importError = " (module not found)";
 			}
 			else
 			{
+				std::string functionName = importName.substr(idxDot + 1);
```

Comment:
Ripping apart the two similar `.substr()` lines worsens readability imho. I assume this was done as an optimization but the else case is taken virtually always as it's the non-error case.